backpack/extensions/secondorder/hbp/__init__.py

```diff
@@ -54,6 +67,8 @@ def __init__(
                 ReLU: activations.HBPReLU(),
                 Sigmoid: activations.HBPSigmoid(),
                 Tanh: activations.HBPTanh(),
+                SumModule: custom_module.HBPSumModule(),
+                Identity: custom_module.HBPScaleModule(),
```

Comment:
Can you also add `ScaleModule: custom_module.HBPScaleModule()` and add one more ResNet setting where `Identity` is replaced by `ScaleModule(weight = xxx)`?

Reply:
Completed this in 
"Reformatting" (1da84d8)